picoquictest/tls_api_test.c

```diff
@@ -5117,7 +5116,7 @@ int virtual_time_test()
         {
             /* Check that the simulated time follows the simulation */
             for (int i = 0; ret == 0 && i < 5; i++) {
-                simulated_time += 12345678;
+                simulated_time += 12345000;
```

Comment:
To have same behaviour as before, may be we should have simulated_time += 12345678000 ?